packages/gatsby-cli/README.md

```diff
@@ -32,10 +32,47 @@ You can also use the `package.json` script variant of these commands, typically
 
 ### `new`
 
-`gatsby new gatsby-site`
+```bash
+gatsby new [<site-name> [<starter-url>]]
+```
+
+#### Arguments
+
+- **`site-name` :** Your Gatsby site name along with the path should be pass here`
```

Comment:
We use tables in the rest of the document so I thought we might want it here too.

```
| Argument | Description |
|-------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| site-name | Your Gatsby site name along with the path should be pass here |
| starter-url | Mention the gatsby starter pack url or local path here. See the [Gatsby starters](https://www.gatsbyjs.org/docs/gatsby-starters/) for more. `Default` starter pack is [gatsby-starter-default](https://github.com/gatsbyjs/gatsby-starter-default). |
```

Reply:
Good idea with the table

Reply:
I had this idea but then I thought that table is for the command options so I dropped it . Ok i will change it !

Reply:
done !